pkg/ebpf/tracer.go

```diff
@@ -448,9 +451,25 @@ func (m *FlowFetcher) LookupAndDeleteMap(met *metrics.Metrics) map[BpfFlowId][]B
 	met.BufferSizeGauge.WithBufferName("hashmap-total").Set(float64(count))
 	met.BufferSizeGauge.WithBufferName("hashmap-unique").Set(float64(len(flows)))
 
+	m.ReadGlobalCounter(met)
 	return flows
 }
 
+// ReadGlobalCounter reads the global counter and updates hashmap update error counter metrics
+func (m *FlowFetcher) ReadGlobalCounter(met *metrics.Metrics) {
+	var allCPUValue []uint32
+	key := uint32(0)
+
+	if err := m.objects.GlobalCounters.Lookup(key, &allCPUValue); err != nil {
+		log.WithError(err).Warnf("couldn't read global counter")
+		return
+	}
+	// aggregate all the counters
+	for _, counter := range allCPUValue {
+		met.Errors.WithErrorName("flow-fetcher", "CannotUpdateHashMapCounter").Add(float64(counter))
```

Comment:
There's already a metric for drops, could we use it instead?

```suggestion
		met.DroppedFlowsCounter.WithSourceAndReason("flow-fetcher", "CannotUpdateHashMapCounter").Add(float64(counter))
```

Reply:
SG will use it then